doc/website/blog/serverless-graphql-using-aws.md

```diff
@@ -0,0 +1,556 @@
+
+import (
+	"database/sql"
+	"entgo-aws-appsync/ent"
+	"entgo-aws-appsync/internal/handler"
+	"log"
+	"os"
+
+	entsql "entgo.io/ent/dialect/sql"
+
+	"entgo.io/ent/dialect"
+	"github.com/aws/aws-lambda-go/lambda"
+	_ "github.com/jackc/pgx/v4/stdlib"
+)
+
+func main() {
+	db, err := sql.Open("pgx", os.Getenv("DATABASE_URL"))
+	if err != nil {
+		log.Fatalf("failed opening database connection: %v", err)
+	}
+
+	client := ent.NewClient(ent.Driver(entsql.OpenDB(dialect.Postgres, db)))
+	defer client.Close()
+
+	lambda.Start(handler.New(client).Handle)
+}
+```
+The function body of `main` is executed whenever an AWS Lambda performs a cold starts.
+After the cold start, a Lambda function is considered "warm," and only the event handler runs, making Lambda executions extremely efficient.
+
+To compile and deploy the Go code, we run:
+```console
+GOOS=linux go build -o main ./lambda
+zip function.zip main
+aws lambda update-function-code --function-name ent --zip-file fileb://function.zip
+```
+The first command creates a compiled binary named `main`.
+The second command compresses the binary to a ZIP archive, required by AWS Lambda.
+The third command replaces the function code of the AWS Lambda named `ent` with the new ZIP archive.
+If you work with multiple AWS accounts you want to use the `--profile <your aws profile>` swiitch.
+
+After you successfully deployed the AWS Lambda, open the "Test" tab of the `ent` function in the web console and invoke it with a "migrate" action:
+<div style={{textAlign: 'center'}}>
+  <img alt="Screenshot of invoking the Ent Lambda with a migrate action" src="https://entgo.io/images/assets/appsync/execution-result.png" />
+  <p style={{fontSize: 12}}>Invoking Lambda with a "migrate" action</p>
+</div>
+On success, you should get a green feedback box and test the result of a "todos" action:
+<div style={{textAlign: 'center'}}>
+  <img alt="Screenshot of invoking the Ent Lambda with a todos action" src="https://entgo.io/images/assets/appsync/execution-result2.png" />
+  <p style={{fontSize: 12}}>Invoking Lambda with a "todos" action</p>
+</div>
+In case the test executions fail, you most probably have an issue with your database connection.
+
+### Configuring AWS AppSync resolvers
+
+With the ent Lambda successfully deployed, we are left to register the ent Lambda as a data source to our AppSync API and configure the schema resolvers to map the AppSync requests to Lambda events.
+First, open our AWS AppSync API in the web console and move to the "data sources" configuration, which you find in the navigation pane on the left.
+<div style={{textAlign: 'center'}}>
+  <img alt="Screenshot of the list of data sources registered to the AWS AppSync API" src="https://entgo.io/images/assets/appsync/data-sources.png" />
+  <p style={{fontSize: 12}}>List of data sources registered to the AWS AppSync API</p>
+</div>
+Click the "Create data source" button in the top right to start registering the Ent function as data source:
```

Comment:
```suggestion
Click the "Create data source" button in the top right to start registering the `ent` function as data source:
```

Reply:
Why would you use `ent` over "ent" here?

Reply:
Because you used `ent` above before.